src/main/java/fr/xephi/authme/AuthMe.java

```
@@ -132,6 +134,12 @@ public void onEnable() {
             stopOrUnload();
             return;
         }
+        
+        // Check if NoCheatPlus is enabled and the fly bug needs to be fixed
+        if (!getServer().getPluginManager().isPluginEnabled("NoCheatPlus")
+                && NoCheatPlusFixSettings.ENABLE_FIX.isValidValue(true)) {
+            ConsoleLogger.warning("You enabled the NoCheatPlus fly kick fix, but NoCheatPlus is not enabled.");
```

can be silently ignored